src/PrestaShopBundle/Resources/config/admin/routing_module.yml

```diff
@@ -24,7 +24,7 @@ admin_module_catalog_refresh:
         _legacy_controller: AdminModules
 
 admin_module_catalog_post:
-    path:     /catalog
+    path:     /catalog/recommended
```

Comment:
could idea, was my mistake to not update it (I wanted to, and I have forgotten to do it :p)